packages/client/src/__test__/client.test.ts

```diff
@@ -1,7 +1,7 @@
 import { rest } from 'msw';
 import { setupServer } from 'msw/node';
 
-import { SKIP_API_URL, SkipRouter } from '../client';
+import { SKIP_API_URL, SkipClient, SkipRouter } from '../client';
```

Comment:
forgot to remove SkipRouter import here

Reply:
on purpose, for testing the deprecated one